Why RawModule instead of Module?

When I compile riscv-boom with the latest hardfloat I have the following problem:

[error] (run-main-0) java.lang.reflect.InvocationTargetException
[error] java.lang.reflect.InvocationTargetException
[error]         at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[error]         at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[error]         at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[error]         at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[error]         at freechips.rocketchip.util.HasGeneratorUtilities.$anonfun$elaborate$1(GeneratorUtils.scala:55)
[error]         at chisel3.Module$.do_apply(Module.scala:52)
[error]         at chisel3.Driver$.$anonfun$elaborate$1(Driver.scala:93)
[error]         at chisel3.internal.Builder$.$anonfun$build$2(Builder.scala:406)
[error]         at scala.util.DynamicVariable.withValue(DynamicVariable.scala:62)
[error]         at chisel3.internal.Builder$.$anonfun$build$1(Builder.scala:404)
[error]         at scala.util.DynamicVariable.withValue(DynamicVariable.scala:62)
[error]         at chisel3.internal.Builder$.build(Builder.scala:404)
[error]         at chisel3.Driver$.elaborate(Driver.scala:93)
[error] hipyard at freechips.rocketchip.util.HasGeneratorUtilities.elaborate(GeneratorUtils.scala:60)
[error] Caused by: chisel3.internal.ChiselException: Error: No implicit clock.
[error] 	at chisel3.internal.throwException$.apply(Error.scala:85)
[error] 	at chisel3.internal.Builder$.$anonfun$forcedClock$1(Builder.scala:319)
[error] 	at scala.Option.getOrElse(Option.scala:189)
[error] 	at chisel3.internal.Builder$.forcedClock(Builder.scala:319)
[error] 	at chisel3.RegInit$.apply(Reg.scala:174)
[error] 	at chisel3.RegInit$.apply(Reg.scala:192)
[error] 	at Chisel.package$Reg$.apply(compatibility.scala:378)
[error] 	at hardfloat.DivSqrtRecF64ToRaw_mulAddZ31.<init>(DivSqrtRecF64_mulAddZ31.scala:85)
[error] 	at hardfloat.DivSqrtRecF64_mulAddZ31.$anonfun$divSqrtRecF64ToRaw$1(DivSqrtRecF64_mulAddZ31.scala:750)
[error] 	at chisel3.Module$.do_apply(Module.scala:52)
[error] 	at hardfloat.DivSqrtRecF64_mulAddZ31.<init>(DivSqrtRecF64_mulAddZ31.scala:750)
[error] 	at hardfloat.DivSqrtRecF64.$anonfun$ds$1(DivSqrtRecF64.scala:60)
[error] 	at chisel3.Module$.do_apply(Module.scala:52)
[error] 	at hardfloat.DivSqrtRecF64.<init>(DivSqrtRecF64.scala:60)
[error] Nonzero exit code: 1

Please explain why the last correction is needed and give advice on how and where it is better to fix it.

--- a/src/main/scala/DivSqrtRecF64.scala
+++ b/src/main/scala/DivSqrtRecF64.scala
@@ -40,7 +40,7 @@ package hardfloat
 import Chisel._
 import consts._
 
-class DivSqrtRecF64 extends chisel3.RawModule
+class DivSqrtRecF64 extends Module
 {
     val io = IO(new Bundle {
         val inReady_div  = Bool(OUTPUT)
@@ -83,7 +83,7 @@ class DivSqrtRecF64 extends chisel3.RawModule
     ds.io.mulAddResult_3 := mul.io.result_s3
 }
 
-class Mul54 extends chisel3.RawModule
+class Mul54 extends Module
 {
     val io = IO(new Bundle {
         val val_s0 = Bool(INPUT)

--- a/src/main/scala/DivSqrtRecF64_mulAddZ31.scala
+++ b/src/main/scala/DivSqrtRecF64_mulAddZ31.scala
@@ -47,7 +47,7 @@ import consts._
 | "docs/DivSqrtRecF64_mulAddZ31.txt" for more details.
 *----------------------------------------------------------------------------*/
 
-class DivSqrtRecF64ToRaw_mulAddZ31(options: Int) extends chisel3.RawModule
+class DivSqrtRecF64ToRaw_mulAddZ31(options: Int) extends Module
 {
     val io = IO(new Bundle {
         /*--------------------------------------------------------------------
@@ -715,7 +715,7 @@ class DivSqrtRecF64ToRaw_mulAddZ31(options: Int) extends chisel3.RawModule
 /*----------------------------------------------------------------------------
 *----------------------------------------------------------------------------*/
 
-class DivSqrtRecF64_mulAddZ31(options: Int) extends chisel3.RawModule
+class DivSqrtRecF64_mulAddZ31(options: Int) extends Module
 {
     val io = IO(new Bundle {
         /*--------------------------------------------------------------------

